here-naksha-lib-model/src/commonMain/kotlin/naksha/model/XyzNs.kt

```diff
@@ -105,7 +107,7 @@ class XyzNs : AnyObject() {
      * Returns the [uuid] as [Guid].
      * @return the [uuid] as [Guid].
      */
-    val guid: Guid = Guid.fromString(uuid)
+    val guid: Guid by lazy { Guid.fromString(uuid) }
```

Comment:
Is there a benefit of lazy init here (e.g. thread-safe)?

Reply:
nope, but it avoids potential overhead - this method is not super simple, let's call only when needed (that's what lazy does - it will get evaluated first time someone actually needs it, then the computed result is stored in memory as part of that object - meaning it won't get calculated again)